services/src/main/java/org/keycloak/protocol/oidc/OIDCLoginProtocolFactory.java

```diff
@@ -110,6 +110,7 @@ public class OIDCLoginProtocolFactory extends AbstractLoginProtocolFactory {
 
```

Comment:
Changed static init of builtIns to be loaded in the init method. This was required as otherwise the Profile wasn't initialised. Regardless we shouldn't have static init code blocks, as that's why we have factories.